apps/judicial-system/web/src/routes/Shared/SignedVerdictOverview/SignedVerdictOverview.spec.tsx

```diff
@@ -38,7 +38,7 @@ describe('Signed Verdict Overview route', () => {
           screen.getByText('Kröfu hafnað', { selector: 'h1' }),
         ),
       ).toBeInTheDocument()
-    })
+    }, 10000)
```

Comment:
This is to [fix unrelated failing tests](https://github.com/island-is/island.is/pull/2628/commits/584fc779affcd66d3e53c80f2de8efe6ca160ac7).